### PR TITLE
add chronosphere partner integration

### PR DESCRIPTION
**Pull request by sophia-hanley**
**Requirements**

- [x] I have followed the repository's [pull request submission guidelines](../blob/main/README.md#submitting-pull-requests)
- [x] I have updated my user documentation
- [x] I have updated my developer documentation (my integration README)
- [x] I have submitted the [Become a Partner](https://launchdarkly.com/partners/) form

**Intended availability**
partner integration

**Related issues**

Provide links to any issues in this repository or elsewhere relating to this pull request.

**Describe the solution you've provided**
Creates a new integration for Chronosphere Change Events, the functionality takes in a secretToken and receiver URL and will work for auditLogEvents

**Describe alternatives you've considered**

If you're modifying an existing integration, provide a clear and concise description of any alternative solutions or features you've considered.

**Provide a request preview**

Include the output from running `npm run preview chronosphere`.
```
URL:    $RECEIVERURL
METHOD: POST
HEADER: Content-Type: application/json
HEADER: X-LD-Signature: $SECRETTOKEN
BODY:	targeting-rule-update.json
{
}
```

**Additional context**

This is a bit hard to test, but I'm expecting the empty template to pass through the body as it is when setting up a normal webhook integration

### CODE REVIEW

File: integrations/chronosphere/manifest.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        ]
+      },
+      "templates": {
+        "default": "templates/default.json.hbs"
+      },
+      "defaultPolicy": [
+        {
+          "effect": "allow",
+          "resources": ["proj/*:env/production:flag/*"],
+          "actions": ["*"]
```
Comment (InTheCloudDan) [inline integrations/chronosphere/manifest.json:56]: @sophia-hanley I'll be pulling this manifest locally to test it but wanted to mentioned with this wildcard for actions, you'll end up receiving a number of change events that are probably not what you're interested in. For example things like tags being added/removed from a flag. They'll have no affect on the runtime.

You can see our suggested list here: https://docs.launchdarkly.com/guides/integrations/build-audit-log-event-hook/?q=building#manifestjson-example though you could even drop the segment section from it.

**Review by InTheCloudDan — CHANGES_REQUESTED** [review]

File: integrations/chronosphere/manifest.json (hunk 1 above)
Comment (sophia-hanley) [reply integrations/chronosphere/manifest.json]: great, I took a look and added the specific actions we're interested in. @InTheCloudDan did this work when you ended up pulling it locally?

**Review by InTheCloudDan — APPROVED** [review]

**Comment by sophia-hanley** [comment]
@InTheCloudDan it looks like i don't have permission to merge this PR, is there anything left to do on my end?

**Comment by InTheCloudDan** [comment]
@sophia-hanley nope, you're all set I'm merging today and will follow up via email with the team.

**Pull request merged**